test/verify/check-system-services

```diff
@@ -288,8 +315,21 @@ Unit=test.service
         b.wait_in_text(self.svc_sel('test-onboot.timer') + ' .service-unit-last-trigger', "unknown")  # last trigger
         self.run_systemctl(user, "stop test-onboot.timer")
 
-        # Selects Paths tab - the test VM does not have any user path units
         if not user:
+            self.goto_service("deleteme.timer")
+            self.do_action("Delete")
+            b.click("#delete-timer-modal-btn")
+            self.wait_page_load()
+            self.assertNotIn("deleteme", m.execute("systemctl list-timers"))
+            self.assertEqual(m.execute(f"! test -f {path}/deleteme.service && echo yes").strip(), "yes")
+
+            # system timers are not allowed to be deleted
+            self.goto_service("cockpit-system.timer")
+            b.click(".service-top-panel .pf-c-dropdown button")
+            b.wait_not_in_text(".service-top-panel .pf-c-dropdown__menu", "Delete")
```

Comment:
Please first wait for an expected action, to make sure the menu has actually opened. Otherwise this is a race condition.